TenureInformationApi.Tests/MockWebApplicationFactory.cs

```diff
@@ -31,6 +31,10 @@ public MockWebApplicationFactory()
             EnsureEnvVarConfigured("DynamoDb_LocalMode", "true");
             EnsureEnvVarConfigured("DynamoDb_LocalServiceUrl", "http://localhost:8000");
 
+            EnsureEnvVarConfigured("AWS_REGION", "eu-west-2");
+            EnsureEnvVarConfigured("AWS_ACCESS_KEY_ID", "local");
+            EnsureEnvVarConfigured("AWS_SECRET_ACCESS_KEY", "local");
```

Comment:
Allows running the tests without Docker - was easier to debug issues

Reply:
How do you launch local dynamo w/o docker?